lib/glific/providers/gupshup/partner/partner_api.ex

```diff
@@ -92,6 +92,24 @@ defmodule Glific.Providers.Gupshup.PartnerAPI do
     end
   end
 
+  @doc """
+  Enable or disable template messaging for an app.
+  """
+  @spec enable_disable_template_messaging(non_neg_integer(), boolean()) :: {:ok, map()} | {:error, any}
+  def enable_disable_template_messaging(org_id, is_hsm_enabled) do
```

Comment:
we only need this to enable template messaging and not for disable template messaging. Can you update the function along with it's name. Make it take only single param